### PR TITLE
Feature/#34 add rating for popular movies fragment and logic for adapter

### PR DESCRIPTION
**Pull request by IstrajI**

### CODE REVIEW

File: ui/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -120,9 +120,7 @@
         <item name="cardPreventCornerOverlap">true</item>
     </style>
 
-
-    <style name="Style.Button.Base" parent="Base.Widget.AppCompat.Button">
-        <item name="colorButtonNormal">?attr/colorAccentButton</item>
+    <style name="RatingBar">
+        <item name="colorControlNormal">#ffd740</item>
```
Comment (mKantsavoi) [inline ui/src/main/res/values/styles.xml]: ?

File: app/src/main/res/layout/adapter_now_playing_movie_item.xml [hunk 2]
```diff
@@ -2,37 +2,48 @@
...
+        android:id="@+id/movie_avr_vote_rating_bar"
+        style="@style/Base.Widget.AppCompat.RatingBar.Small"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="bottom|start"
+        android:layout_marginBottom="@dimen/default_content_spacing"
+        android:layout_marginLeft="@dimen/spacing_40"
+        android:numStars="5"
+        android:stepSize="0.5"
+        android:theme="?attr/style.RatingBar" />
```
Comment (mKantsavoi) [inline app/src/main/res/layout/adapter_now_playing_movie_item.xml]: why set attr like theme?

File: app/src/main/res/layout/adapter_now_playing_movie_item.xml [hunk 3]
```diff
@@ -2,37 +2,48 @@
...
+        android:textAppearance="?attr/textAppearance.Body1.Primary" />
 
+    <android.support.v7.widget.AppCompatRatingBar
+        android:id="@+id/movie_avr_vote_rating_bar"
+        style="@style/Base.Widget.AppCompat.RatingBar.Small"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="bottom|start"
+        android:layout_marginBottom="@dimen/default_content_spacing"
+        android:layout_marginLeft="@dimen/spacing_40"
```
Comment (mKantsavoi) [inline app/src/main/res/layout/adapter_now_playing_movie_item.xml]: will be better set textview and ratingbar inside linear

File: ui/src/main/res/values/styles.xml (hunk 1 above)
Comment (IstrajI) [reply ui/src/main/res/values/styles.xml]: They aren't connected. Just delet one code and write another on same place.
I will change color to reference of color

Comment (mKantsavoi) [reply ui/src/main/res/values/styles.xml]: Hm, strangely

**Pull request merged**